README.md

```diff
@@ -192,8 +192,7 @@ updateUserConfiguration(`{
 }`)
 ```
 
-### Troubleshoot
```

Comment:
@CGNonofr I removed this headline, because it is confusing, because there is a "Troubleshooting" section below. Hope you are ok with it.

Reply:
Thanks! :)